test/behavior.zig

```diff
@@ -128,4 +128,23 @@ test {
     {
         _ = @import("behavior/export_keyword.zig");
     }
+
+    comptime {
+        if (builtin.link_libc and supportsTermios()) {
+            _ = @import("behavior/termios_compat.zig");
+        }
+    }
+}
+
+fn supportsTermios() bool {
+    // isBSD covers all Darwin and BSD variants.
+    if (builtin.os.tag.isBSD()) return true;
+
+    // Solaris and Illumos.
+    if (builtin.os.tag.isSolarish()) return true;
+
+    return switch (builtin.os.tag) {
+        .linux, .haiku, .wasi, .emscripten => true,
```

Comment:
This was an attempt to shorten the list instead of reproducing what's in std.
I couldn't find a better way—as far as I can tell, if I try to reference tc_lflag_t, it'll be a compiler error on platforms where it isn't supported, so I can't just check for its existence.